content/en/continuous_integration/setup_tests/python.md

```diff
@@ -11,25 +11,18 @@ further_reading:
 ---
 ## Compatibility
 
-Supported Test Frameworks:
-* Pytest 3.0+
-
-Supported CI providers:
-* Appveyor
-* Azure Pipelines
-* BitBucket
-* BuildKite
-* CircleCI
-* GitHub Actions
-* GitLab
-* Jenkins
-* TravisCI
+Supported Python interpreters:
+* Python >= 2.7 and >= 3.5
+
+Supported test frameworks:
+* pytest >= 3.0.0
+  * pytest < 5 when using Python 2
 
 ## Prerequisites
 
 [Install the Datadog Agent to collect tests data][1].
 
-## Installing tracing
+## Installing the Python tracer
 
 Install the Python tracer by running:
```

Comment:
ℹ️  rc5 exists now. Not sure if we want people using that version instead

Reply:
It doesn't come with anything that we need, so we can wait until the stable version is released to update this.